common/lib/xmodule/xmodule/tabs.py

```diff
@@ -471,14 +470,18 @@ def iterate_displayable(course, user=None, inline_collections=True, include_hidd
     @classmethod
     def upgrade_tabs(cls, tabs):
         """
-        Reverse and Rename Courseware to Course and Course Info to Home Tabs.
+        Remove course_info tab, and rename courseware tab to Course if needed
         """
         if tabs and len(tabs) > 1:
+            # Reverse them so that course_info is first, and rename courseware to Course
             if tabs[0].get('type') == 'courseware' and tabs[1].get('type') == 'course_info':
                 tabs[0], tabs[1] = tabs[1], tabs[0]
-                tabs[0]['name'] = _('Home')
                 tabs[1]['name'] = _('Course')
 
+            # If course_info tab is present, it should be at index 0. If so, remove it
+            if tabs[0].get('type') == 'course_info':
+                tabs.pop(0)
+
         return tabs
```

Comment:
Just wanted to explicitly call out this change. Originally, I was going to add logic to `validate_tabs` to ignore the `course_info` tab, but not actually modify any of the contents of the tabs passed to it. Then I remembered that this method actually makes modifications to the tab list, and from its name, seems to be designed to upgrade old tab definitions. I decided to keep the existing code in case a course still has the older definition, but I removed the modification to the `course_info` tab's name because I also subsequently am removing that tab from the list. I also [added tests](https://github.com/edx/edx-platform/blob/1ae1f76b9728c52841e12c767441b26a9bf31fbb/common/lib/xmodule/xmodule/tests/test_tabs.py) that ensure correctness for both `validate_tabs` and `upgrade_tabs`.

Reply:
@pcockwell, this approach looks reasonable to me, without this we wouldn't be able to import older courses :+1: 

Could you please add some info to the docstring of `upgrade_tabs` explaining that this is related to a removal of unified course tabs as a part of TNL-7061?